errors.ts

```diff
@@ -99,6 +83,26 @@ export function installUncaughtExceptionListener(actionOnException?: () => void)
 	});
 }
 
+function tryTrackException(error: Error, injector: IInjector): void {
+	let disableAnalytics: boolean;
+	try {
```

Comment:
I think setting disableAnalytics to a default value belongs to the static config and not here.